content/hi/docs/setup/best-practices/node-conformance.md

```diff
@@ -0,0 +1,88 @@
+नोड कुबेरनेट्स के लिए न्यूनतम आवश्यकताओं को पूरा करता है; एक नोड जो परीक्षा पास करता है
+कयूबेर्नेतेस क्लस्टर में शामिल होने के लिए योग्य है।
+
+## नोड पूर्वापेक्षा
+
+नोड अनुरूपता परीक्षण चलाने के लिए, एक नोड को उसी पूर्वापेक्षाओं को पूरा करना चाहिए जैसे 
+मानक कुबेरनेट्स नोड। कम से कम, नोड में निम्नलिखित होना चाहिए
+डेमॉन स्थापित:
+
+* कंटेनर रनटाइम (डॉकर)
+* क्यूबलेट
+
+## नोड अनुरूपता परीक्षण कैसे चलाएं
+
+नोड अनुरूपता परीक्षण चलाने के लिए, निम्न चरणों का पालन करें:
+1. क्यूबलेट के लिए `--kubeconfig` विकल्प के मान की गणना करें; उदाहरण के लिए:
+    `--kubeconfig=/var/lib/kubelet/config.yaml`।
+    क्योंकि परीक्षण ढांचा क्यूबलेट का परीक्षण करने के लिए एक स्थानीय कण्ट्रोल प्लेन शुरू करता है,
+         API सर्वर के URL के रूप में `http://localhost:8080` का उपयोग करें।
+     कुछ अन्य क्यूबलेट कमांड लाइन पैरामीटर हैं जिनका आप उपयोग करना चाह सकते हैं:
+   * `--pod-cidr`: यदि आप `kubenet` का उपयोग कर रहे हैं, तो आपको एक मनमाना CIDR निर्दिष्ट करना चाहिए
+     क्यूबलेट के लिए, उदाहरण के लिए `--pod-cidr=10.180.0.0/24`।
+   * `--cloud-provider`: यदि आप `--cloud-provider=gce` का उपयोग कर रहे हैं, तो आपको यह करना चाहिए
+     परीक्षण चलाने के लिए ध्वज को हटा दें।
+
+2. आदेश के साथ नोड अनुरूपता परीक्षण चलाएँ:
+
+```shell
+# $CONFIG_DIR आपके क्यूबलेट का पॉड मेनिफेस्ट पथ है।
+# $LOG_DIR परीक्षण आउटपुट पथ है।
+sudo docker run -it --rm --privileged --net=host \
+  -v /:/rootfs -v $CONFIG_DIR:$CONFIG_DIR -v $LOG_DIR:/var/result \
+  k8s.gcr.io/node-test:0.2
+```
+
+## अन्य आर्किटेक्चर के लिए नोड अनुरूपता परीक्षण चलाना
+
+कुबेरनेट्स अन्य वास्तुकला के लिए नोड अनुरूपता परीक्षण डॉकर चित्र भी प्रदान करता है:
+
+ Arch  |       Image       |
+--------|:-----------------:|
+ amd64  |  node-test-amd64  |
+  arm   |    node-test-arm  |
+ arm64  |  node-test-arm64  |
+
+## चयनित परीक्षण कैसे चलाएं 
+
+विशिष्ट परीक्षण चलाने के लिए, पर्यावरण चर `FOCUS` को उन परीक्षणों की नियमित अभिव्यक्ति के साथ अधिलेखित करें, जिन्हें आप चलाना चाहते हैं।
+
+```shell
+sudo docker run -it --rm --privileged --net=host \
+  -v /:/rootfs:ro -v $CONFIG_DIR:$CONFIG_DIR -v $LOG_DIR:/var/result \
+  -e FOCUS=MirrorPod \ # केवल मिररपॉड टेस्ट चलाएं
+  k8s.gcr.io/node-test:0.2
+```
+
+विशिष्ट परीक्षणों को छोड़ने के लिए, पर्यावरण चर `SKIP` को उन परीक्षणों की नियमित अभिव्यक्ति के साथ अधिलेखित करें जिन्हें आप छोड़ना चाहते हैं।
+
+
+```shell
+sudo docker run -it --rm --privileged --net=host \
+  -v /:/rootfs:ro -v $CONFIG_DIR:$CONFIG_DIR -v $LOG_DIR:/var/result \
+   -e SKIP=MirrorPod \ # सभी अनुरूपता परीक्षण चलाएं लेकिन मिररपॉड परीक्षण को छोड़ दें
+   k8s.gcr.io/node-test:0.2
+```
+
+नोड अनुरूपता परीक्षण [node e2e test](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-node/e2e-node-tests.md) का एक कंटेनरीकृत संस्करण है।
+डिफ़ॉल्ट रूप से, यह सभी अनुरूपता परीक्षण चलाता है।
+
+सैद्धांतिक रूप से, आप किसी भी नोड e2e परीक्षण को चला सकते हैं यदि आप कंटेनर को कॉन्फ़िगर करते हैं और
+आवश्यक वॉल्यूम ठीक से माउंट करें। लेकिन **केवल अनुरूपता चलाने की पुरज़ोर अनुशंसा की जाती है
```

Comment:
Hi , in the following sentence, `लेकिन **केवल अनुरूपता चलाने की पुरज़ोर अनुशंसा की जाती है परीक्षण ** ` : 
the placement of word `परीक्षण` should be directly after `अनुरूपता` and not at end of sentence.
as the words `conformance test` together translate to अनुरूपता परीक्षण.